web/viewer.css

```diff
@@ -1428,10 +1429,33 @@ html[dir='rtl'] .outlineItemToggler::before {
 #passwordOverlay > .dialog {
   text-align: center;
 }
+
 #passwordOverlay .toolbarField {
   width: 200px;
 }
 
+@media print {
+  #mozPrintCallback-shim {
+    display: none;
+    page-break-after: always;
+    page-break-inside: avoid;
+  }
+}
+
+#mozPrintCallback-shim .progress-row {
+  clear: both;
+  padding: 1em 0;
+}
+
+#mozPrintCallback-shim progress {
+  width: 100%;
+}
+
+#mozPrintCallback-shim .relative-progress {
+  clear: both;
+  float: right;
+}
```

Comment:
Given that the `mozPrintCallback` is only included in `!(FIREFOX || MOZCENTRAL)` builds, I don't that we should place styles specific to it in `viewer.css` since those rules will be include for *all* builds. Hence I'd suggest that you move the above rules back.